frontend/src/components/chats/components/chats-list/components/chat/styles.module.scss

```diff
@@ -39,6 +39,10 @@
 }
 
 .lastMessage {
+  max-width: 320px;
```

Comment:
Can't we replace it with something like 100%?

Reply:
https://gist.github.com/elky/ce230d6fcab055104cdf1a3358037fc7